### PR TITLE
chore: update branding settings to link to new form

### LINKED ISSUE #1961: New branding request: Forms is missing an H1
The request form is set with an H2 instead of an H1

The confirmation page is also missing an H1, no screenshot is available.

### LINKED ISSUE #2892: Update branding request intake form content
Branding request forms have been requiring a lot of back-and-forth between support and the client. 

Looking to update the content and form to help reduce this effort. 

### Acceptance criteria:
- [x] Standardize language (branding v logo)
- [x] Add content to let people know which applications we'll accept
~ - [ ] Restrict file type to SVG ~
- [x] Review tone (e.g., "building a library")
- [x] Add a field around naming convention for the dropdown (right now, the "alt text" field dictates the naming within the dropdown, which is a different use case).
  - [x] Change the "corporate logo Health Canada" drop-down to "Health Canada"

### PR DESCRIPTION
**Pull request by anikbrazeau**
Fixes #2892 
Fixes #1961

### CODE REVIEW

**Comment by anikbrazeau** [comment]
Need help to make right link show up in EN / FR 

EN: Submit alternate branding https://articles.alpha.canada.ca/forms-formulaires/request-alternate-branding
FR: Demander une autre image de marque https://articles.alpha.canada.ca/forms-formulaires/fr/demander-une-autre-image-de-marque/

File: public/static/locales/en/form-builder.json [hunk 1]
```diff
@@ -209,11 +209,11 @@
...
     "heading": "Branding",
-    "notFound": "Didn’t find your department or agency on the list?",
-    "preview": "Preview of branding",
-    "select": "Select a different branding option for the top of your form",
-    "submitNew": "Submit a new branding option (opens in new tab)",
-    "text1": "GC Forms displays the Government of Canada signature at the top of your form by default."
+    "notFound": "Didn’t find your department or agency in the list?",
+    "preview": "Preview",
+    "select": "Select different branding for the top of your form",
+    "submitNew": "Submit alternate branding",
```
Comment (anikbrazeau) [inline public/static/locales/en/form-builder.json:215]:
```suggestion
    "submitNew": "Request alternate branding",
```

File: public/static/locales/fr/form-builder.json [hunk 2]
```diff
@@ -210,10 +210,10 @@
     "default": "(par défaut)",
     "heading": "Image de marque",
     "notFound": "Vous ne trouvez pas votre ministère ou votre organisme dans la liste?",
-    "preview": "Prévisualisation de l'image de marque",
-    "select": "Sélectionnez l’image de marque de votre formulaire",
-    "submitNew": "Proposer une nouvelle option d’image de marque (ouvre un nouvel onglet)",
-    "text1": "Formulaires GC affiche par défaut la signature du gouvernement du Canada dans l’en-tête de votre formulaire."
+    "preview": "Aperçu",
+    "select": "Sélectionnez une image de marque différente pour le haut du formulaire",
+    "submitNew": "Proposer une nouvelle option d'image de marque",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:215]:
```suggestion
    "submitNew": "Demander une autre image de marque",
```

**Review by thiessenp-cds — APPROVED** [review]
Nice work. LGTM :)

**Pull request merged**